### DISCUSSION TITLE
Brew install TailwindCSS Standalone CLI

### DISCUSSION THREAD

**Discussion by ccostello97** [tailwindlabs/tailwindcss#11956] [Ideas]
Would like to see a Homebrew Formula/Cask for the TailwindCSS Standalone CLI.

(I tried submitting one myself, but I abandoned the effort.)

Still, I think it would be good to boost adoption and make it cleanly installable/uninstallable and auto-update-able.

Does anyone else see the value in this?

**Comment by SHi-ON** [comment]
It's available now as:
```bash
brew install aureuma/tailwindcss/tailwindcss-standalone
```
